src/main/scala/exercises/03_Exponent.scala

```diff
@@ -14,6 +14,10 @@ object Exponent {
    * NOTE: Scala has a built-in exponent function, but you're not allowed to
    * use it to implement `expt`
    */
-  def expt(n: Int, e: Int): Int = -1
+  def expt(n: Int, e: Int): Int = {
+      if(e == 0) 1
+      else if(n == 0) 0
```

Comment:
I think you'll be fine without this case.  It will just multiple by zero in the else case which will guarantee a return result of 0.